src/Page/HelpSelf.elm

```diff
@@ -31,10 +34,59 @@ update msg model =
 view : Model -> Html Msg
 view model =
     div []
-        [ div [] [ text "Help Self page - grid and singles" ]
+        [ header []
+            [ h1 [ css [ pageHeadingStyle ] ] [ text "[cCc] Help Me With.." ]
+            ]
+        , ul [ css [ gridStyle ] ]
+            [ infoLink "COVID-19" "#"
+            , infoLink "Understanding Economic Abuse" "#"
+            , infoLink "Banking" "#"
+            , infoLink "Debt" "#"
+            , infoLink "Housing" "#"
+            ]
         , ul []
             [ li [] [ a [ href (t GetHelpPageSlug) ] [ text "Get Help" ] ]
             , li [] [ a [ href (t NotAlonePageSlug) ] [ text "Read about others" ] ]
             , li [] [ a [ href (t DefinitionPageSlug) ] [ text "What is Economic Abuse" ] ]
             ]
         ]
+
+
+infoLink : String -> String -> Html msg
+infoLink title link =
+    li
+        [ css
+            [ cardStyle
+            , withMedia [ only screen [ Media.minWidth (px 576) ] ]
+                [ flex3 zero zero threeColumn ]
+            ]
+        ]
+        [ a [ css [ blockLinkStyle ], href link ] [ div [] [ text title ] ]
+        ]
+
+
+blockLinkStyle : Style
+blockLinkStyle =
+    batch
+        [ color colours.purple
+        , displayFlex
+        , flexDirection column
+        , fontFamilies [ "Raleway", sansSerif.value ]
+        , fontSize (rem 1.25)
+        , Css.height (pct 100)
+        , justifyContent center
+        , textDecoration none
+        , Css.width (pct 100)
+        ]
+
+
+cardStyle : Style
+cardStyle =
+    batch
```

Comment:
I guess there might be potentially other attributes we want to adjust in the media query once the real style get up - so could be good to have `desktopCardStyle` and `mobileCardStyle`if that happens to make it more clear.